XLS-0085d-token-escrow/README.md

```diff
@@ -0,0 +1,231 @@
+# XLS-85d Token Escrow — Token-Enabled Escrows
+
+```markdown
+Title: Token-Enabled Escrows
+Revision: 7 (2025-02-17)
+Type: Draft
+Author:
+    Denis Angell, XRPL-Labs [dangell7](https://github.com/dangell7)
+Affiliation: XRPL-Labs
+Contributors:
+    Richard Holland, XRPL-Labs [RichardAH](https://github.com/RichardAH)
+    Ed Hennis, Ripple [ximinez](https://github.com/ximinez)
+    Mayukha Vadari, Ripple [msvadari](https://github.com/msvadari)
```

Comment:
...did I contribute to this spec?

Reply:
I went ahead and just removed the "contributors"